### DISCUSSION TITLE
Horizontal scrollbar is not visible

### DISCUSSION THREAD

**Discussion by ericnewcomer** [nyaruka/floweditor#1151] [Ideas]
Scrolling works, but you need a trackpad to get it done.

**Comment by putorti** [comment]
Going to add, if this is the same issue, that `overflow-x:hidden` on body makes it impossible to scroll horizontally on the contacts screen if the user has more featured fields than fit, the featured fields seem to have a min-width. The touchpad or mouse doesn't solve this particular issue.
